e2e/actions/oci.go

```diff
@@ -23,6 +23,18 @@ import (
 	"gotest.tools/v3/assert"
 )
 
+const (
+	imgTestFilePath    string = "file-for-testing"
+	squashfsTestString string = "squashfs-test-string"
+	extfsTestString    string = "extfs-test-string"
+)
+
+var (
+	imgsPath        = filepath.Join("..", "internal", "pkg", "util", "fs", "overlay", "testdata")
```

Comment:
If we're using from the e2e and unit test we should move these file under the top level `test/` directory. I believe there's a place there for fs images already (SIFs used in various tests).